.github/workflows/website.yml

```diff
@@ -0,0 +1,27 @@
+name: Website
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Probably because of this check?

Reply:
I think I was trying to keep this behavior 👇 

https://github.com/oblador/loki/blob/4be00352e8554d7aff69c49f670fc247a9d22e6e/.travis.yml#L40

Reply:
I'll remove that since it doesn't make sense to require a branch here

Reply:
Fixed by https://github.com/oblador/loki/pull/375